.circleci/config.yml

```diff
@@ -516,12 +516,12 @@ workflows:
           <<: *filters_all_branches_and_tags
       - orb/build_and_test_integration:
           name: build_and_test_integration-2.3
-          integration_apps: 'rack rails-five'
+          integration_apps: 'rack'
```

Comment:
There was an issue with Nokogiri. I tried fixing it, but there was some incompatibility issue due to the ruby version. 
https://www.fastruby.io/blog/ruby/rails/versions/compatibility-table.html